src/app/shared/cookies/browser-klaro.service.spec.ts

```diff
@@ -11,15 +11,28 @@ import { CookieService } from '../../core/services/cookie.service';
 import { getTestScheduler } from 'jasmine-marbles';
 import { MetadataValue } from '../../core/shared/metadata.models';
 import { cloneDeep } from 'lodash';
+import { ConfigurationDataService } from '../../core/data/configuration-data.service';
+import { createSuccessfulRemoteDataObject$ } from '../remote-data.utils';
+import { ConfigurationProperty } from '../../core/shared/configuration-property.model';
 
 describe('BrowserKlaroService', () => {
```

Comment:
Can you add some new tests to cover the "check configuration / filter services" logic?